src/DynamoCore/Search/NodeAutocompleteSearch.cs

```diff
@@ -35,49 +37,52 @@ internal MLNodeAutoCompletionRequest(string dynamoVersion, int numberOfResults)
         internal IEnumerable<PackageItem> Packages { get; set; }
 
         [DataMember(Name = "context")]
-        internal ContextRequest Context { get; set; }
+        internal ContextItem Context { get; set; }
     }
 
+    /// <summary>
+    /// Data representing node properties for ML autocomplete.
+    /// </summary>
     [DataContract]
-    internal class NodeRequest
+    public class NodeItem
```

Comment:
If it's internal visible, do we still need to make it public?

Reply:
Actually this object was being used in the NodeAutocompleteCluster(this had to be public) which is exposed in the view extension. But for now I can make it internal as we are not previewing nodes/connectors. Can make it public later if needed.